back/src/chat/chat.gateway.ts

```diff
@@ -18,7 +12,7 @@ import Socket from '@type/socket';
 		credentials: true
 	}
 })
-export class ChatGateway implements OnGatewayInit, OnGatewayConnection, OnGatewayDisconnect {
```

Comment:
Why is OnGatewayDisconnect no longer needed?

Reply:
Yes, we don't need it anymore, as the sole purpose was to update the clientMap. Now, as we're using socket.io rooms, everything is handled by socket.io